addons/modules/XEH_postInit.sqf

```diff
@@ -81,3 +81,9 @@ if (isServer) then {
     _object setDir _yaw;
     [_object, _pitch, _roll] call BIS_fnc_setPitchBank;
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(crewToGunner), {
```

Comment:
`moveToGunner` might be a better event name.

Reply:
https://github.com/zen-mod/ZEN/pull/423/commits/c30e30d5c25cc995c705c6484dde9fe9a9c7599a